src/main/java/teammates/logic/core/CoursesLogic.java

```diff
@@ -722,4 +722,13 @@ public boolean hasIndicatedSections(String courseId) throws EntityDoesNotExistEx
         return archivedCourseIds;
     }
 
+    public String validateIsInstructorDisplaytoStudents(List<InstructorAttributes> instructors, String email) {
```

Comment:
Add a header comment for this method.

Also, the way I see the method is used, it could be better as just a boolean method `isAtLeastOneInstructorDisplayedToStudents`.